core/imodel-bridge/src/IModelBridge.ts

```diff
@@ -28,34 +28,35 @@ export abstract class IModelBridge {
   }
 
   /** This is only called the first time this source data is synchronized.  Allows the bridge to perform any steps after the Job Subject has been created.  It
-   * must call synchronizer.recordDocument on the source data.
+   * must call synchronizer.recordDocument on the source data. Called in the [Repository channel]($docs/learning/backend/channels).

```

Comment:
@swwilson-bsi Are these links intended to point to Channel.md directly, or does something map `channels` to that file?

Reply:
It is supposed to link to channels.md.

Reply:
Do you know if doc links are case-sensitive ("Channel" vs "channel")? @williamkbentley?

Reply:
The $docs link syntax is case insensitive.

Reply:
Fixed.